models/cable/R/write_restart.CABLE.R

```diff
@@ -0,0 +1,18 @@
+#' @title Write restart template for SDA
+#' 
+#' @author Alexey Shiklomanov
+#' 
+#' @param start.time   Time of current assimilation step 
+#' @param stop.time    Time of next assimilation step
+#' @param new.state    Analysis state matrix returned by \code{sda.enkf}
+#' @inheritParams read.restart.CABLE
+#' 
+#' @description Write restart files for model
+#' 
+#' @export
+write_restart.CABLE <- function(outdir,
+                                    runid,
+                                    start.time,
+                                    stop.time,
+                                    settings,
+                                    new.state) {}
```

Comment:
As with other unimplemented functions, this and read_restart should probably throw a logger.sever